testsuite/MDAnalysisTests/topology/test_mmtf.py

```diff
@@ -33,6 +33,20 @@ class TestMMTFParser_gz(TestMMTFParser):
     expected_n_segments = 4
 
 
+class TestMMTFSkinny(ParserBase):
```

Comment:
Does it test `ts.dimensions`?

Does it test what happens if we try to access an optional topology attribute?

Reply:
dimensions tests now in coordinates/ & added tests for missing MMTF attributes